account/manager_test.go

```diff
@@ -1430,3 +1430,28 @@ func TestMakeTxnLabel(t *testing.T) {
 		require.Equal(t, genLabel, testCase.label)
 	}
 }
+
+// TestParseTxLabel tests whether an example account transaction label can be
+// parsed correctly.
+func TestParseTxLabel(t *testing.T) {
+	t.Parallel()
+
+	label := `poold -- {"account":{"action":"withdraw","key":` +
```

Comment:
The label currently isn't written in this format (see comment above). But given the test assumes it is, we should probably fix that in the code where we create the label.